Formula/dockutil.rb

```diff
@@ -14,13 +14,65 @@ class Dockutil < Formula
     sha256 cellar: :any_skip_relocation, all: "f5f87d9e286c2b294bb157ac9f87baa2720fff044c7a92c0b80b9cb82db8a87e"
   end
 
+  depends_on xcode: ["12.5", :build]
   depends_on :macos
 
+  # Create Package.swift
```

Comment:
This should be submitted upstream

Reply:
There is no problem building dockutil upstream. Problem, as I [explained](https://github.com/Homebrew/homebrew-core/pull/97394#issuecomment-1075676550] is within Homebrew. Therefore in order to workaround these problems, the patch was created. If you believe this is unacceptable, then I see only one way of handling this upgrade of dockutil:

1. Remove formulae for dockutil from homebrew-core.
1. Add formulae for dockutil into homebrew-cask.

Reply:
Since there hasn't been any upstream response and this is currently non-functional on 12.3 without a `python`, should we consider merging either this patch or the one from the pending PR https://github.com/kcrawford/dockutil/pull/131?